src/api/app/views/webui/action_seen_by_users/toggle_action_seen_by_user.js.erb

```diff
@@ -0,0 +1,7 @@
+$('#action-seen-by-user-wrapper').html(
+  "<%= escape_javascript(render ActionSeenByUserComponent.new(action: @action, user: @user)) %>"
+);
+
+$('#action-seen-by-user-wrapper-in-select-<%= @action.id %>').html(
+  "<%= escape_javascript(render ActionSeenByUserComponent.new(action: @action, user: @user, render_only: true)) %>"
+);
```

Comment:
Something is wrong with your editor, you're missing a newline at the end of this file, see https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
Fixed both, the file and my editor's rule. Thanks for the pointer :+1: